surveymonkey/api_surveymonkey.py

```diff
@@ -53,15 +54,25 @@ def __call_api_post(self, url, data):
         This uses the session to make a POST request and returns the response
         """
         response = self.session.post(url=url, json=data)
-        LOG.info("Surveymonkey post response with status code = %s", response.status_code)
+        LOG.info("Surveymonkey post response with status code = %s %s", response.status_code, url)
         return response
 
     def __call_api_get(self, url, payload):
         """
         This uses the session to make a GET request and return the response
         """
         response = self.session.get(url=url, params=payload)
-        LOG.info("Surveymonkey get response with status code = %s", response.status_code)
+        LOG.info("Surveymonkey get response with status code = %s %s", response.status_code, url)
+        return response
+
+    def _patch_api_call(self, url, payload):
```

Comment:
```suggestion
    def __call_api_patch(self, url, payload):
```
in order to keep the same notation

Reply:
Why does the notation is with double underscore? @andrey-canon Just asking...

Reply:
https://docs.python.org/3/tutorial/classes.html#private-variables

Reply:
Addressed, but you are not using it for the purpose for which it was made.